Phase01Project1/src/SearchFilters/SearchFilter.java

```diff
@@ -0,0 +1,21 @@
+package SearchFilters;
+
+import java.util.ArrayList;
+import java.util.Map;
+import Documents.Document;
+
+public abstract class SearchFilter {
```

Comment:
SearchFilter isn't the right name. it does the filter so something like "Filtering" in it?
just a suggestion